eng/targets/Imports.targets

```diff
@@ -102,10 +102,19 @@
 
   <!-- Rulesets -->
   <PropertyGroup Condition="'$(CodeAnalysisRuleSet)' == ''">
-    <CodeAnalysisRuleSet Condition="'$(IsShipping)' == 'true'">$(RepositoryEngineeringDir)config\rulesets\Shipping.ruleset</CodeAnalysisRuleSet>
-    <CodeAnalysisRuleSet Condition="'$(IsShipping)' != 'true'">$(RepositoryEngineeringDir)config\rulesets\NonShipping.ruleset</CodeAnalysisRuleSet>
+    <CodeAnalysisRuleSet>$(RepositoryEngineeringDir)config\rulesets\Empty.ruleset</CodeAnalysisRuleSet>
```

Comment:
Why?

Reply:
It wasn't clear what the ramifications would be if no ruleset was specified.

Reply:
@jmarolf Made the change to remove the logic in Microsoft.CodeAnalysis.targets to remove `CodeAnalysisRuleSet` being defaulted when unset. I think removing this property group completely should work - otherwise we have a product issue to address as we cannot expect customers to create and set empty rulesets.